src/js/components/EditsTable.jsx

```diff
@@ -52,8 +50,17 @@ export const renderTableCaption = (edit, rowObj, pagination) => {
   const editText = length === 1 ? 'edit' : 'edits'
   const captionHeader = `${length} ${name} ${editText} found.`
 
+  if(type === 'macro') {
+    return (
+      <div className="caption">
+        <h3>{captionHeader}</h3>
+        {description ? <p className="usa-font-lead">{description}</p>:null}
+      </div>
+    )
+  }
+
   return (
-    <caption>
+    <caption className="caption">
```

Comment:
Ok, I'll merge because I'd like to have this as part of my pagination updates but would rather not add unused classes. It can be a bit misleading when looking at/for the CSS.